dartagnan/src/main/java/com/dat3m/dartagnan/verification/RefinementTask.java

```diff
@@ -123,7 +123,7 @@ private Wmm createDefaultWmm() {
 	        repo.addRelation(porfco);
 	        Relation porfcofr = new RelUnion(porfco, fr);
 	        repo.addRelation(porfcofr);
-	        baseline.addAxiom(new Acyclic(porfcofr));
+	        baseline.addAxiom(new Acyclic(porfcofr, false, false));
```

Comment:
It might be better to keep the simple constructor as well as the new complex constructor. I don't have a strong opinion about this though.